open-api/rest-catalog-open-api.yaml

```diff
@@ -4817,6 +4955,16 @@ components:
           }
         }
 
+    IdempotencyInProgressError:
```

Comment:
Sorry if this has been discussed before, but a "InProgressError" doesn't sound quite right to me? I understand there's a window but wouldn't it be some sort of "RequestAlreadyProcessed" (past tense)?

Reply:
My bad, this is the case for detecting a duplicate request for one that is already in flight. If the request is already successfully processed, nothing to do but return a sucessfull response.